sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -340,6 +341,7 @@ case class FileSourceScanExec(
         "PartitionFilters" -> seqToString(partitionFilters),
         "PushedFilters" -> seqToString(pushedDownFilters),
         "DataFilters" -> seqToString(dataFilters),
+        "BucketedScan" -> bucketedScan.toString,
```

Comment:
this breaks some explain tests, shall we do it later?

Reply:
ok